src/objective-c/GRPCClient/private/GRPCWrappedCall.m

```diff
@@ -263,7 +264,7 @@ - (instancetype)initWithHost:(NSString *)host
     // queue. Currently we use a singleton queue.
     _queue = [GRPCCompletionQueue completionQueue];
 
-    _call = [[GRPCHost hostWithAddress:host] unmanagedCallWithPath:path completionQueue:_queue];
+    _call = [[GRPCHost hostWithAddress:host] unmanagedCallWithPath:path authority:authority completionQueue:_queue];
```

Comment:
since `authority` is `nonnull`, it should be included in the existence check above